From 80abdea43418361bf7f14cb52bb35bc7dec8bba3 Mon Sep 17 00:00:00 2001
From: Christopher Feghali <christopher.feghali.xb@renesas.com>
Date: Thu, 30 Apr 2020 15:54:24 +0100
Subject: [PATCH] remove cwd from lib_path to make it relative for
 cross-compilation

Prevents appending "=" to -L which then appends target sysroot to the
library directory.

Signed-off-by: Christopher Feghali <christopher.feghali.xb@renesas.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7352d3b667..c48e48b400 100644
--- a/setup.py
+++ b/setup.py
@@ -233,7 +233,7 @@ else:
 
 # Constant known variables used throughout this file
 cwd = os.path.dirname(os.path.abspath(__file__))
-lib_path = os.path.join(cwd, "torch", "lib")
+lib_path = os.path.join("torch", "lib")
 third_party_path = os.path.join(cwd, "third_party")
 caffe2_build_dir = os.path.join(cwd, "build")
 # lib/pythonx.x/site-packages
-- 
2.17.1

